packages/dashboard-core-plugins/src/ConsolePlugin.tsx

```diff
@@ -78,6 +80,40 @@ export function ConsolePlugin(
     [panelManager]
   );
 
+  const addOpenFileMapEntry = useCallback(
```

Comment:
Yoooo great work! Good catch, didn't even realize we were mutating the `openFileMap`/`previewFileMap` before.

Reply:
Yes, it was doing something weird where the console was logging it with length 0 but it had entries.